README.md

```diff
@@ -93,25 +94,23 @@ use alexandria::math::sha512::sha512;
 ### Build
 
 ```bash
-make build
+scarb build
 ```
 
-### Run
+### Test
 
 ```bash
-make run
+scarb test
 ```
-
-### Test
-
+Running a specific subset of tests 
 ```bash
-make test
+scarb test -f math
 ```
 
 ### Format
 
 ```bash
-make format
+scarb format
```

Comment:
If you wish to format cairo files only:
```suggestion
scarb fmt
```

If you wish to use the script from Scarb manifest: 
```suggestion
scarb run format
```